email/1.3/modules/ROOT/pages/email-gmail.adoc

```diff
@@ -1,30 +1,49 @@
 = Connect to Gmail with Email Connector Examples - Mule 4
 :page-aliases: connectors::email/email-gmail.adoc
 
-Configure Anypoint Connector for Email (Email Connector) to connect to your Gmail account.
+To connect Anypoint Connector for Email (Email Connector) to your Gmail account, enable *Less Secure Apps* in your account, and also configure a global element for the server connection IMAPS, POP3S, or SMTPS to use, both in Anypoint Studio and XML.
```

Comment:
I suggest: ...enable *Less Secure Apps* in your account and configure a global element...
This might apply to the other PR as well - do you need "both in Anypont Studio and XML" text? It is not clear what that is referring to.